recipes/pep8/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.7.0" %}
+
+package:
+  name: pep8
```

Comment:
@ocefpaf I can do the rename and then add it. I was having issues with rebasing to fix the notes from @jakirkham though; my plan was to clean everything up by deleting the branch and going from scratch. Does that seem better?